upgrade-2.x.sh

```diff
@@ -1291,6 +974,23 @@ FETCHED_SLUG=$(CURL_CA_BUNDLE="${TMPCRT}" ${CURL} -H "Authorization: Bearer ${AP
 SLUG=${FORCED_SLUG:-$FETCHED_SLUG}
 HOST_OS_VERSION=${META_BALENA_VERSION:-${VERSION_ID}}
 
+# Check host OS version
+case $VERSION in
+    [2-9].*|2[0-9][0-9][0-9].*.*)
+        if ! version_gt "$target_version" "$minimum_hostos_version" &&
+            ! [ "$target_version" == "$minimum_hostos_version" ]; then
```

Comment:
Isn't this v simpler than two negations?
```suggestion
        if ! version_gt "$minimum_hostos_version" "$target_version"; then
```

Reply:
Oh I also think you meant to be checking he host OS version here instead of the target one
```suggestion
        if ! version_gt "$VERSION" "$minimum_hostos_version" &&
            ! [ "$VERSION" == "$minimum_hostos_version" ]; then
```
to be clear I'm fine with ^ as well if you prefer to just stick with the way we do the target version check for consistency.